src/components/rangeFilterInput/index.stories.mdx

```diff
@@ -0,0 +1,105 @@
+import { useState } from 'react';
+import { Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import { Box } from '@chakra-ui/react';
+
+import RangeFilterInput from './index';
+
+<Meta title="Components/Forms/FilterInput" component={RangeFilterInput} />;
```

Comment:
There is already a group for form/filters:
![Screenshot 2023-02-21 at 11 25 23](https://user-images.githubusercontent.com/144707/220318797-7890aed9-ead4-400d-b91e-0293a515683c.png)

Could you use that one?

Reply:
Missed that somehow :(